config/routes.rb

```diff
@@ -14,6 +14,9 @@
     resources :basic_projects, path: 'basic-projects'
     resources :calendar, only: [:index]
     resources :calendar_events, only: [:index]
+    scope '(:type)' do # parens lets this param be optional
```

Comment:
I could not find a way to have the param come after data_exports AND have path helpers etc work

Reply:
We discussed this and I sent you an alternate route definition we can use here. 

```ruby
resources :data_exports, path_names: { new: '(/:type)/new' }
```